src/realm.js

```diff
@@ -173,6 +174,7 @@ export class Realm {
   timeout: void | number;
   mathRandomGenerator: void | (() => number);
   strictlyMonotonicDateNow: boolean;
+  maxStackDepth: void | number;
```

Comment:
Seems that "maxStackDepth" is never undefined? If so, you want to remove "void" from the typing.

Reply:
fixed